.gitignore

```diff
@@ -5,3 +5,4 @@ source/*/*.min.js.map
 test/results
 /dev.sh
 .DS_Store
+.idea
```

Comment:
I do, but IntelliJ IDEA is one of the most popular IDE, strange that you did not know :)